assets/install-rhcos.sh

```
@@ -1,53 +1,145 @@
+        fi
+
+        efibootmgr -c -d $device -p 2 -l '\EFI\redhat\shimaa64.efi' -L "Red-Hat CoreOS GRUB"
+
+    else
+        ilog "INFO: Ignition file is missing, skipping installation."
+        exit 1
+    fi
+}
+
+# Firmware related identification
+cx_pcidev=$(lspci -nD 2> /dev/null | grep 15b3:a2d[26c] | awk '{print $1}' | head -1)
```

Shellcheck (0.10.0)

[warning] Quote the grep pattern so the shell won't interpret it.

(SC2062)